hyperware-ai/hyperdrive
# Fix osname ownership check UI

## Problem

- When a user is registering an osname, there's no message in the UI if a name is already claimed.

## Solution

- Our ethname hashing library is out of date, switch to a working fork under a different organization (`@ensdomains`)
- Polyfill node `Buffer` to make the above work in-browser

## Testing

- In a new node, try to register an existing name
- Confirm the "Name is already claimed" message displays
- Try to register an unclaimed name
- Confirm you can start up the new node as expected
